index.php

```diff
@@ -23,7 +23,8 @@
 <a href="http://github.com/lmanul/whylinuxisbetter">GitHub</a> and come contribute!</p>
 
 <p>Have a look at <a href="switch/try/oneday/index.php">One day with Linux (a screenshot tour)</a>!</p>
-
+<p>Note: The following features are not specific to just Linux, 

```

Comment:
I disagree, its about a simple note to the reader that these features are not specific to GNU/Linux and there are other alternatives(BSD, Solaris, etc).

I think my above post justifies why this was a correct approach.